lib/mysql.js

```diff
@@ -127,21 +121,16 @@ require('util').inherits(MySQL, SqlConnector);
  * @param {String} sql The SQL statement
  * @param {Function} [callback] The callback after the SQL statement is executed
  */
-MySQL.prototype.query = function (sql, callback) {
+MySQL.prototype.executeSQL = function (sql, params, options, callback) {
```

Comment:
This is confusing...? Can't we use `SQLConnector#execute()`. If this is mysql specific - maybe we should name the method so that is clear.